packages/web-components/src/components/rux-clock/rux-clock.scss

```diff
@@ -71,10 +68,12 @@
 .rux-clock__segment__label {
     font-size: 0.875rem;
     color: var(--clock-label-color);
-}
-
-.rux-clock__day-of-the-year .rux-clock__segment__value {
-    border-right: none;
+    background: var(--clock-border-color);
+    border: 1px solid var(--clock-border-color);
+    width: -moz-available;
+    width: -webkit-fill-available;
+    width: fill-available;
```

Comment:
does width: 100% not work?

Reply:
It extends it like 1 or 2 px past the date and time. And 99% is barley too short